user-guide-chapters/introduction/launching.adoc

```diff
@@ -36,18 +43,22 @@ A simple JSON configuration file is shown in the following example.
 }
 ----
 
-The first attribute of the JSON data is the distribution specification version
+The configuration file contains the distribution specification version
 (`distSpecVersion`). The structure and content of other attributes are described
 in the later sections of this guide.
 
-Using the information in this guide, compose a configuration file with the
+Using the information in this guide, you can compose a configuration file with the
 settings and features you require for your {zotLowerName} registry server.
+
```

Comment:
Added a tip: "When deploying a full {zotLowerName} image, you can enable or disable extensions individually. With a minimal image, omit the extensions section."

Reply:
Even if extensions section is present, it will be ignored. So language can say, configuration need not include extensions and if it does, will be ignored.

`
{"level":"warn","goroutine":1,"caller":"zotregistry.io/zot/pkg/extensions/minimal.go:51","time":"2022-06-01T16:56:18.704462615Z","message":"skipping setting up extensions routes because given zot binary doesn't support any extensions, please build zot full binary for this feature"}
`